packages/boot/test/bootstrapTests/test-addAssets.ts

```diff
@@ -0,0 +1,196 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import type { TestFn } from 'ava';
+
+import { TimeMath } from '@agoric/time';
+import {
+  LiquidationTestContext,
+  makeLiquidationTestContext,
+} from './liquidation.ts';
+
+const test = anyTest as TestFn<LiquidationTestContext>;
+
+const auctioneerPath = 'published.auction';
+
+test.before(async t => {
+  t.context = await makeLiquidationTestContext(t);
+  const proposal = await t.context.buildProposal({
+    package: 'builders',
+    packageScriptName: 'build:add-STARS-proposal',
+  });
+
+  t.log('installing proposal');
+  // share a single proposal so tests don't stomp on each other's files; It has
+  // to be edited by each so as not to re-use keywords.
+  for await (const bundle of proposal.bundles) {
+    await t.context.controller.validateAndInstallBundle(bundle);
+  }
+  t.log('installed', proposal.bundles.length, 'bundles');
+  // @ts-expect-error override
+  t.context.proposal = proposal;
+  t.log('installed', proposal.bundles.length, 'bundles');
+});
+
+test.after.always(t => {
+  // This will fail if a subset of tests are run. It detects that three
```

Comment:
Why do we need this check? Is each test not independent?

Reply:
Actually, now that I moved building the proposal into `.before`, the total time was 38s. When I modified the tests to `.serial` that grew to 40s, which is probably not significant.

Reply:
not significant. also probably a smaller difference in CI with fewer cores.